### ISSUE TITLE
SoundCloud requests update to Google Play Services

### ISSUE THREAD

**Issue by jackturnbull** [microg/android_packages_apps_GmsCore#152]
migrog 0.2.3, SoundCloud 2016.06.28-release

SoundCloud still works as you'd expect (does without microg) although I get the following dialog box:

```
Update Google Play Services

SoundCloud won't run unless you update Google Play services.
```

Screen:

<img src="https://cloud.githubusercontent.com/assets/3958636/16724708/01f49ad0-474c-11e6-9d7f-4a7a35ecf0e6.png" width="320" />

**Comment by mar-v-in** [comment]
Please try with latest preview release 0.2.3-6 (download from drop-down [here](https://microg.org/download.html) or the f-droid repo).

**Comment by jackturnbull** [comment]
Doesn't appear to be showing on my f-droid, looks like the microG app listing has disappeared.

**Comment by ImAnnoying** [comment]
It appears as ug UnifiedNlp (no GAPPS), but it's actually microg

**Comment by jackturnbull** [comment]
Fixed in 0.2.3-6-gbee7bc9
